.github/workflows/packaging.yml

```
@@ -58,7 +58,7 @@ jobs:
           with open('.github/data/distros.yml') as f:
               data = yaml.load(f)
 
-          if "${{ github.event_name }}" == "pull_request" and "${{ contains(github.event.pull_request.labels.*.name, 'ci/packaging') }}" != "true":
+          if "${{ github.event_name }}" == "pull_request" and "${{ contains(github.event.pull_request.labels.*.name, 'run-ci/packaging') }}" != "true":
```

```suggestion
          if "${{ github.event_name }}" == "pull_request" and "${{ !contains(github.event.pull_request.labels.*.name, 'run-ci/packaging') }}":
```
this is to please reviewdog (151 > 150 :smile:)
[! means not](https://docs.github.com/en/actions/learn-github-actions/expressions#operators)